### PR TITLE
resolve issue #56

### PR DESCRIPTION
**Pull request by wangchen615**
Added documentation and manifests to deploy load watcher as a service.

### CODE REVIEW

File: pkg/watcher/internal/metricsprovider/prometheus.go
Comment (lenhattan86) [inline pkg/watcher/internal/metricsprovider/prometheus.go:102]: why do we use log.Fatalf but not return error? I think we should make consistent across the metrics clients.

**Review by lenhattan86 — CHANGES_REQUESTED** [review]

File: pkg/watcher/internal/metricsprovider/prometheus.go
Comment (wangchen615) [reply pkg/watcher/internal/metricsprovider/prometheus.go:102]: Yes, @lenhattan86 , you are right. I am going to change it. Still working on it and will share when I am ready.

**Comment by wangchen615** [comment]
@lenhattan86 , I have fixed the problem you raised. Could you have another review and merge?

**Review by lenhattan86 — APPROVED** [review]
looks good to me now.

**Comment by lenhattan86** [comment]
@wangchen615 feel freel to merge it.

**Comment by wangchen615** [comment]
@lenhattan86 could you merge on your side? my squash and merge button is grayed as it is pending your /approve

**Pull request merged**
